server/cmd/server/main.go

```diff
@@ -5,8 +5,8 @@ import (
     "log"
     "os"
     "net/http"
-	"server/pkg/http/rest"
-	"server/pkg/storage/pg"
+	"server/internal/http/rest"  // changes made here for new path to rest and pg folder
```

Comment:
I suggest removing this comment since it is only relative to the backend restructure branch & not master itself.

Reply:
Agreed